docs/rofl/deployment.md

```diff
@@ -7,249 +7,24 @@ all of the required functionality.
-By default, the resulting file will be called `rofl-oracle.orc`.
-
-:::info Reproducibility
-
-For audit reasons it is very important that ROFL app binaries can be reproduced
-from the given source code. This makes it possible to check that the right code
-is actually deployed. In order to support reproducible builds, please see the
-[Reproducibility chapter].
-
-[Reproducibility chapter]: https://github.com/oasisprotocol/oasis-sdk/blob/main/docs/runtime/reproducibility.md
-
-:::
-
-## Updating the ROFL App Policy
-
-Now that the app binaries are available, we need to update the policy with the
-correct cryptographic identity of the app. To obtain the identity of the app
-that was just built run:
-
-```shell
-oasis rofl identity rofl-oracle.orc
-```
-
-This should output something like the following:
-
-```
-0+tTmlVjUvP0eIHXH7Dld3svPppCUdKDwYxnzplndLea/8+uR7hI7CyvHEm0soNTHhzEJfk1grNoBuUqQ9eNGg==
-```
-
-This represents the cryptographic identity of the ROFL app. We now need to
-update the policy to ensure that only exact instances of the built app can
-successfully authenticate under our app ID. To do so, update the previously
-generated `policy.yml` as follows (using your own app identity):
-
-<!-- markdownlint-disable line-length -->
-![code yaml {10-12}](../../examples/runtime-sdk/rofl-oracle/policy2.yml "policy.yml")
-<!-- markdownlint-enable line-length -->
-
-Then to update the on-chain policy, run (using _your own app identifier_ instead
-of the placeholder `rofl1qqn9xndja7e2pnxhttktmecvwzz0yqwxsquqyxdf`):
-
-```shell
-oasis rofl update rofl1qqn9xndja7e2pnxhttktmecvwzz0yqwxsquqyxdf \
-  --policy policy.yml \
-  --admin self \
-  --network testnet \
-  --paratime sapphire
-```
-
-:::info
-
-For those interested in the details, the specified cryptographic identity is
-actually based on the MRENCLAVE and MRSIGNER pair used in Intel SGX remote
-attestation. Because the ROFL protocols only use MRENCLAVE for authentication,
-a random signer key is generated during build and used to sign the enclave.
-
-:::
-
-## Deploying the ROFL App
+## Deploying the ROFL App Bundle
 
 ROFL apps are deployed through Oasis nodes running on systems that support the
-targeted TEE (e.g. Intel SGX). If you don't have a running node where you could
+targeted TEE (e.g. Intel TDX). If you don't have a running node where you could
 deploy your ROFL app, please first make sure that you have a client node with
 the Sapphire Testnet runtime configured (see the [client node documentation] for
-instructions on setting one up). Note that you need at least version 24.2 of
+instructions on setting one up). Note that you need at least version 25.0 of
 Oasis Core.
 
-<!-- TODO: Make it really simple to spin up new nodes using checkpoints. -->
-<!-- TODO: Include some reasonable pruning defaults. -->
-
-After your node is set up, update the `runtime` section in your configuration
-as follows:
+After your node is set up, include the built `myapp.default.orc` in the
```

Comment:
Maybe a link/reference to the section in which we build the app would be nice here (https://github.com/oasisprotocol/oasis-sdk/blob/535d2da4b22452367277ce0a38b48e8d32e17079/docs/rofl/app.mdx#build)